sms_plusserver.py

```diff
@@ -214,13 +226,19 @@ def send(self, sms, timeout=None, fail_silently=False):
             result = bool(put_response and put_response.is_ok)
         return result
 
-    def check_state(self, sms, wait=False, timeout=None, fail_silently=False):
+    def check_state(
+        self,
+        sms: 'SMS',
```

Comment:
Let's add `from __future__ import annotations` and use `sms: SMS` here.

Reply:
Same as here:
https://github.com/W-Z-FinTech-GmbH/sms_plusserver/pull/6#discussion_r670384862